source/patterns/01-molecules/inline-list/inline-list.twig

```diff
@@ -0,0 +1,39 @@
+{%- import 'atoms-html' as html -%}
+
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['inline-list']) }) -%}
+
+{%- if small|default(false) -%}
+  {%- set attributes = attributes|merge({class: attributes.class|merge(['inline-list--small']) }) -%}
+{%- endif -%}
+
```

Comment:
@thewilkybarkid I think we need something like this to address https://github.com/libero/pattern-library/pull/110/files#r293713491
```suggestion
{%- if semiColonSeparated|default(false) -%}
  {%- set attributes = attributes|merge({class: attributes.class|merge(['inline-list--semi-colon-separated']) }) -%}
{%- endif -%}
```

Reply:
Maybe `alternativeSeparators`?

Reply:
Adjusted name to `alternativeSeparators`

Reply:
Added in https://github.com/libero/pattern-library/pull/108/commits/7308885962cbe79295caf2796e8a6d6899431e56.